packages/vscode-extension/src/project/metro.ts

```diff
@@ -292,6 +292,82 @@ export class Metro implements Disposable {
     ws.close();
   }
 
+  private async evaluateJsFromCdp(
+    webSocketDebuggerUrl: string,
+    source: string,
+    timeoutMs: number = 2000
+  ): Promise<string | undefined> {
+    const REQUEST_ID = 0;
```

Comment:
I wonder if there is a reason why part of the variables is declared in the promise and rest outside, is there any reason?

Reply:
if there is no reason to be defined inside the promise as for example websocket connection needs to be opened inside i lest it outside and it seems a good pattern. (I moved timeoutHandle declaration inside the promise now as it was only an empty declaration)